src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java

```diff
@@ -1,40 +1,129 @@
 package edu.kis.vh.nursery.list;
 
+/**
+* Linked list storing integers
+*
+* */
 public class IntLinkedList {
 
-    Node last;
-    int i;
+    private static final int EMPTY = -1;
+    private Node last;
```

Comment:
'last' shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internallt usually do not require encapsulation.